iothub/service/src/ServiceClient.cs

```diff
@@ -215,6 +213,8 @@ public static ServiceClient CreateFromConnectionString(string connectionString,
 
 #endif
 
+        internal IotHubConnection Connection { get; }
```

Comment:
properties before methods?

Reply:
Yes, but I'd say not before static factory methods, which are more like ctors. Would you agree?